### DISCUSSION TITLE
Update concourse/dex dependancy to at least v2.39.0 from upstream

### DISCUSSION THREAD

**Discussion by Kump3r** [concourse/concourse#8961] [Help & Support] [answered]
Hello all,
With the following [PR](https://github.com/dexidp/dex/pull/2454), we contributed to the [dexidp/dex](https://github.com/dexidp/dex.git), in order to log failed authentication requests, which is a security requirement to have a mechanism of coping with brute force attacks. I am opening this discussion and I have also written a comment in #5525 and in [discord thread](https://discord.com/channels/219899946617274369/413771057686773770/1242064682345435146). Following [this documentation](https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md#rebuilding-to-test-your-changes), I was able to validate successfully, that the upstream change, will log such attempts in the web logs in the future. Can you provide an estimate as to when a release, containing dex upstream changes from [v2.39.0](https://github.com/dexidp/dex/tree/v2.39.0), would be available for Concourse? Thanks in advance!

**Comment by taylorsilva** [comment, marked as answer]
Hey, someone needs to update our fork here: https://github.com/concourse/dex

PR's are welcome. Going to v2.39.0 seems like a safe path.

**Reply by taylorsilva** [reply]
Since this is actually a unit of work I'm going to start an issue.
